packages/apollo-server/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "apollo-server",
+  "private": true,
+  "version": "2.0.0-beta.0",
+  "description": "Production ready GraphQL Server",
+  "author": "opensource@apollographql.com",
+  "main": "dist/index.js",
+  "scripts": {
+    "compile": "rimraf dist && tsc",
+    "deploy": "cd npm && npm publish --tag=beta",
+    "postcompile": "rimraf npm && mkdir npm && mv dist/* npm/. && ./scripts/prepare-package.sh",
+    "predeploy": "npm run compile",
+    "prepublish": "echo '\n\nPlease use npm run deploy to publish apollo-server\n\n' && exit 1",
+    "watch": "tsc -w"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/apollographql/apollo-server/tree/master/packages/apollo-server"
+  },
+  "keywords": [
+    "GraphQL",
+    "Apollo",
+    "Server",
+    "Javascript"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/apollographql/apollo-server/issues"
+  },
+  "homepage": "https://github.com/apollographql/apollo-server#readme",
+  "devDependencies": {
+    "@koa/cors": "^2.2.1",
+    "@types/cors": "^2.8.3",
+    "@types/express": "^4.11.1",
+    "@types/graphql": "^0.12.7",
+    "@types/koa": "^2.0.45",
+    "@types/koa-bodyparser": "^4.2.0",
+    "@types/koa-router": "^7.0.27",
+    "@types/koa__cors": "^2.2.2",
+    "@types/node": "^9.6.1",
+    "@types/react": "^16.3.8",
+    "@types/ws": "^4.0.2",
+    "apollo-cache-control": "^0.1.1",
+    "apollo-engine": "^1.0.6",
+    "rimraf": "^2.6.2",
+    "typescript": "2.8.1"
+  },
+  "peerDependencies": {
+    "graphql": "^0.12.0 || ^0.13.0 || ^14.0.0"
+  },
+  "typings": "dist/index.d.ts",
+  "dependencies": {
+    "apollo-server-core": "^1.4.0-alpha.0",
+    "apollo-server-express": "^1.3.4",
+    "apollo-server-koa": "^1.3.4",
+    "body-parser": "^1.18.2",
+    "cors": "^2.8.4",
+    "express": "^4.16.3",
+    "graphql-disable-introspection": "^1.0.1",
+    "graphql-playground-middleware-express": "^1.6.1",
+    "graphql-playground-middleware-hapi": "^1.6.1",
+    "graphql-playground-middleware-koa": "^1.5.1",
+    "graphql-playground-middleware-lambda": "^1.5.1",
+    "graphql-subscriptions": "^0.5.8",
+    "graphql-tools": "^2.23.1",
+    "koa-bodyparser": "^4.2.0",
```

Comment:
@glasser I don't think this would prevent hapi16 and hapi17 from both being packages considering the fact that no frameworks are direct dependents of the middleware packages [for example, hapi](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-server-hapi/package.json#L27-L31).

It is unfortunate that all middleware packages will be included, however they are relatively small and even with related dependencies not an overall large footprint.

The advantage of having the packages under the apollo-server namespace is a delineation between the utility driven, do it yourself model of the current middleware, and the opinionated take this package is looking to provide to people adopting GraphQL

Reply:
Ah, OK, I guess it's just that koa happens to be an outlier with dependencies you need in order to use its middleware (though arguably Express has the same issue with body-parser).

I just am not super sure that the fine distinction between `apollo-server/express` and `apollo-server-express` will actually be appreciated by our users? It seems like it might be confusing. And adding those extra dependencies to the package that we want to encourage most people to use (depending on packages totally irrelevant to their use case) seems like a shame. 🤷‍♂️  if you're convinced then I trust you. It's just making less sense to me than it did a few weeks ago.

I guess one nice upside is that `apollo-server-foo` *doesn't* have these extra dependencies.